src/util/Injected.js

```diff
@@ -106,6 +351,71 @@ exports.LoadUtils = () => {
 
     };
 
+    window.WWebJS.prepareMessageButtons = (buttonsOptions) => {
+        const returnObject = {};
+        if (!buttonsOptions.buttons) {
+            return returnObject;
+        }
+        
+        returnObject.title = buttonsOptions.title;
+        returnObject.footer = buttonsOptions.footer;
+    
+        if (buttonsOptions.useTemplateButtons) {
+            returnObject.isFromTemplate = true;
```

Comment:
ignore this line to fix it for ios

Reply:
@PurpShell

Reply:
that will remove URL & Call features, and will also invalidate the UI view.